modules/data-entry/src/main/frontend/src/questionnaire/FormPagination.jsx

```diff
@@ -213,15 +220,15 @@ function FormPagination (props) {
   }, [saveInProgress, pendingSubmission, disableProgress, nextActivePage, direction, activePage]);
 
   useEffect(() => {
-    if (activePage != null && pages != null) {
+    if (activeStep != null && numberOfSteps != null) {
```

Comment:
```suggestion
    if (activeStep != null && numberOfSteps > 0) {
```
^ Still ensures numberOfSteps is defined and also avoids division by 0